src/static/css/page.css

```
@@ -196,6 +196,11 @@
   font-weight: normal;
 }
 
+.byline .author,
+.byline .reviewer {
+  white-space: nowrap;
+}
+
```

We should include analysts and reviewers in this too.

Should we just change this to the generic `.byline a`?:

```suggestion
.byline a {
  white-space: nowrap;
}

```